src/Calculator.js

```diff
@@ -0,0 +1,46 @@
+export default class Calculator {
+
+    calculate(input) {
+        if (!input) return 0
+
+        const delimiters = this.getDelimiters(input)
+        const tokens = this.splitByDelimiters(input, delimiters)
+        this.isValidInput(tokens)
+
+        return tokens.reduce((acc, cur) => acc + cur, 0)
+
+
+    }
+
+    getDelimiters(input) {
+        const defaulDelimiters = [",", ":"]
+        let customDelimiter = input.match(/\/\/(.*?)\\n/g);
+
+        if (customDelimiter) {
+            customDelimiter = customDelimiter.map(item => item.slice(2, -2)).filter(delimiter => delimiter !== "")
+        }// //와 \n 사이에 아무것도 들어오지 않을 경우 기본 구분자를 사용하도록
+
+        return customDelimiter ? [...defaulDelimiters, ...customDelimiter] : defaulDelimiters
+    }
```

Comment:
if 조건문으로 `customDelimiter`가 있을 때를 한번 감쌌는데 아래에서 `customDelimiter` 기준으로 또 삼항연산자로 나누니깐 저는 코드 흐름이 약간 어색해지는 느낌을 받은 것 같네요

코드를 전체적으로 봤을 때 주희님이 어떤 의도를 가지셨는지 알 수 있지만 조금 더 헷갈리지 않게 개선할 수 있는 방법이 있지 않을까? 생각해보게 되는 것 같아요🤔

```suggestion
    getDelimiters(input) {
        const defaulDelimiters = [",", ":"]
        const matchDelimiter = input.match(/\/\/(.*?)\\n/g);
        const customDelimiter = matchDelimiter?.map(item => item.slice(2, -2)).filter(delimiter => delimiter !== "")

        if (customDelimiter) { // 커스텀 구분자가 있으면 기본 구분자 + 커스텀 구분자 리턴
            return  [...defaulDelimiters, ...customDelimiter]
        }

        return defaulDelimiters // 기본 구분자만 리턴
    }
```
이런 느낌으로 계산을 위로 몰아서 아래쪽에는 조건에 따라 결과를 리턴해주는 방식은 어떨까요??